.github/workflows-off/slither.yml

```diff
@@ -0,0 +1,43 @@
+name: Slither Analysis
```

Comment:
I would only add the command. Think that it is not necessary run it  in all commit